<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::focusingcontrollerwidget</class>
 <widget class="QWidget" name="snowgui::focusingcontrollerwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="1" column="3">
    <widget class="QSpinBox" name="stepsizeSpinBox">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="repositoryLabel">
     <property name="text">
      <string>Repository:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="solverLabel">
     <property name="text">
      <string>Solver method:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="centerLabel">
     <property name="text">
      <string>Center:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="evaluationLabel">
     <property name="text">
      <string>Evaluation method:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="stepsLabel">
     <property name="text">
      <string>Steps:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="evaluationBox"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="stepsizeLabel">
     <property name="text">
      <string>Step size:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="stepsSpinBox">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="repositoryBox"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="solverBox"/>
   </item>
   <item row="0" column="5">
    <widget class="QSpinBox" name="centerSpinBox">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>214748367</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="value">
      <number>8000000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="6" rowspan="2">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
